mfa/methods/code/messaging.md

```diff
@@ -83,6 +89,8 @@ request.setApplicationId(applicationId);
 request.setFrom(fromNumber);
 request.setTo(toNumber);
 request.setScope("scope");
+request.setDigits(5);
+request.setMessage("Your temporary {NAME} {SCOPE} code is {CODE}");
```

Comment:
You are just intending that this be a string with the literal `{NAME}` correct?

Reply:
Yeah that's what its usage is